config/sitemap.rb

```diff
@@ -76,6 +76,12 @@
   add edit_registration_role_type_path
   add edit_registration_role_type_other_path
 
+  # close account
+  add edit_reason_user_close_account_path
+  add confirm_close_account_user_close_account_path
```

Comment:
We could make this action just `confirm` to make it something more human readable.